charts/codesealer/values.yaml

```diff
@@ -107,6 +107,9 @@ webhook:
 
   # affinity: {}
 
+sidecar:
+  enabled: false
```

Comment:
Can't comment on that line, but on line 606 there's still a `manager.enabled = false`. I suppose that should be removed?